src/sentry/search/events/datasets/discover.py

```diff
@@ -1020,6 +1024,18 @@ def function_converter(self) -> Mapping[str, SnQLFunction]:
                     ),
                     private=True,
                 ),
+                SnQLFunction(
+                    "salted_column_hash",
+                    # TODO: figure out how to make this take arbitrary number of arbirary type arguemnts
+                    required_args=[SnQLStringArg("salt"), ColumnArg("column")],
+                    snql_aggregate=lambda args, alias: Function(
+                        "farmFingerprint64",  # farmFingerprint64 aka farmHash64 is a newer, faster replacement for cityHash64
+                        [args["salt"], args["column"]],
```

Comment:
Out of curiosity, do we actually need a salt here? Can we just hash the id column alone?

Reply:
That is a good question. The options are:
- use `event.id` as is. It's a UUID, so in theory it's already randomized if we sort by it;
- use hash of just  the `event.id`; 
- use hash with a non-static salt (more random?);

So, is it strictly needed? I guess not really if we're dealing with data that has real event IDs generated as UUID. I guess I auto-suggested myself with test data that re-used same event IDs across multiple projects. I'll simplify this.

Reply:
I've removed the salt, but kept hashing event.id